packages/cli/src/platform/project/create.ts

```diff
@@ -1,13 +1,25 @@
 import { arg, Command, isError } from '@prisma/internals'
+import { log } from 'console'
```

Comment:
If you could avoid this and use `console.info`, we don't use this at all in the codebase and it's better to be consistent.